opentelekomcloud/resource_opentelekomcloud_waf_domain_v1.go

```diff
@@ -120,40 +135,63 @@ func resourceWafDomainV1() *schema.Resource {
 	}
 }
 
-func getAllServers(d *schema.ResourceData) []domains.ServerOpts {
+func firstOneSet(res map[string]interface{}, k1, k2 string) (interface{}, error) {
```

Comment:
maybe move it to helpers?

Reply:
This method is specific for string values, as values are compared with `""`. Not  sure if it can be moved to helpers

Reply:
i think in other resources we can face same situation as here, but it's up to you